pkg/justification/processor.go

```diff
@@ -68,14 +69,19 @@ const (
 	DefaultAudience = "dev.abcxyz.jvs"
 )
 
+func (p *Processor) WithValidators(v map[string]jvspb.Validator) *Processor {
```

Comment:
nit: comment

Also, thinking one step ahead. Maybe we want to expose a func to return the validators. In the `controller`, we need the validators to populate categories (and later hints, etc.)

Reply:
Ah, yeah, plan to do that in the next PR when I am adjusting the controller to expose the category in UI. Will resolve this thread now